hi-ml-histopathology/environment.yml

```diff
@@ -11,46 +11,31 @@ dependencies:
   - python-blosc==1.7.0
   - torchvision=0.11.1
   - pip:
+      - -r ../test_requirements.txt
```

Comment:
shouldn't be the other way round? I think test_requirements should include all the requirements of the package plus optionally other packages required for testing. 
I think a developer will naturally update the package environment if something new is needed while developing and with the current setup it will not be ensured the tests will also update the environment

Reply:
or maybe I don't understand correctly what test_requirements.txt is used for? if it's more of a generic hi-ml package requirement file maybe we can rename it?